go/vt/vttablet/tabletmanager/rpc_vreplication.go

```diff
@@ -282,7 +487,7 @@ func (tm *TabletManager) UpdateVReplicationWorkflow(ctx context.Context, req *ta
 			tabletTypes = req.TabletTypes
 		}
 		tabletTypesStr := topoproto.MakeStringTypeCSV(tabletTypes)
-		if inorder && req.TabletSelectionPreference == tabletmanagerdatapb.TabletSelectionPreference_UNKNOWN ||
+		if (inorder && req.TabletSelectionPreference == tabletmanagerdatapb.TabletSelectionPreference_UNKNOWN) ||
```

Comment:
should we group the other half of the `||` in parens as well (for clarity/symmetry)?